### ISSUE TITLE
Parameter 'style' has incorrect value 'undefined'

### ISSUE THREAD

**Issue by laur4c** [cdimascio/express-openapi-validator#473]
Hello,

It seems like the **style** parameter is required but this keywork has **form** as default value
https://swagger.io/docs/specification/serialization/#query

Why it's required and the validation fail instead of using the spec default value?

E.g:

While trying to use the following JSON schema to define a query parameter:

```
{
  "description":"Description",
  "required":false,
  "schema":{
    "title":"State",
    "type":"array",
    "items":{
      "$ref":"#/components/schemas/ThreadState"
    },
    "description":"A description"
  },
  "name":"state",
  "in":"query"
}
```

I get the error:
```
{
    "message": "Parameter 'style' has incorrect value 'undefined' for [state]",
    "errors": [
        {
            "path": ".query.state",
            "message": "Parameter 'style' has incorrect value 'undefined' for [state]"
        }
    ]
}
```

Any thoughts? Maybe I'm missing something

Thanks!

**Comment by cdimascio** [comment]
thanks for the issue @laur4c. it is fixed in v4.7.2

**Comment by laur4c** [comment]
Thanks for the fix!
